### PR TITLE
build for ios

### PR DESCRIPTION
**Pull request by cbodonnell**
Total work in progress as this point.  Goal is to get this automated before merging anything.  It depends on my credentials at the moment, but want to get this PR up so you can take a look.

### CODE REVIEW

File: dg-app-expo/.gitignore [hunk 1]
```diff
@@ -4,5 +4,5 @@
 # @end expo-cli 
 expo-env.d.ts
 
-# local env files
-.env*.local
+# env files
+.env*
```
Comment (cbodonnell) [inline dg-app-expo/.gitignore:8]: this kept overriding the env vars for the build. even when specifying build-time vars in the `eas.json` file

File: dg-app-expo/app.json [hunk 2]
```diff
@@ -16,13 +16,18 @@
       "**/*"
     ],
     "ios": {
-      "supportsTablet": true
+      "supportsTablet": true,
+      "bundleIdentifier": "com.cbodonnell.dgappexpo",
+      "buildNumber": "5"
```
Comment (cbodonnell) [inline dg-app-expo/app.json:21]: `buildNumber` needs to be incremented for each build

File: dg-app-expo/eas.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      "developmentClient": true,
+      "distribution": "internal"
+    },
+    "preview": {
+      "distribution": "internal"
+    },
+    "production": {
+      "env": {
+        "EXPO_PUBLIC_API_IP": "https://dg-app-backend.tunnel.farm"
+      }
```
Comment (cbodonnell) [inline dg-app-expo/eas.json:16]: This section allows for setting env vars based on the build configuration.

File: dg-app-expo/package.json [hunk 4]
```diff
@@ -8,7 +8,9 @@
     "ios": "expo start --ios",
     "web": "expo start --web",
     "test": "jest --watchAll",
-    "postinstall": "node postinstall.js"
+    "postinstall": "node postinstall.js",
+    "build:ios": "EXPO_NO_DOTENV=1 eas build --platform ios",
+    "submit:ios": "EXPO_NO_DOTENV=1 eas submit -p ios"
```
Comment (cbodonnell) [inline dg-app-expo/package.json:13]: Now that I think of it, using `EXPO_NO_DOTENV=1` here might allow us to keep tracking the base `.env` file if we want.  I'll have to test this.

File: dg-app-expo/.gitignore (hunk 1 above)
Comment (cbodonnell) [reply dg-app-expo/.gitignore]: not needed with a top-level easignore (until we find something better to do)

**Review by cstrab — APPROVED** [review]

**Pull request merged**